internal/common/test_configuration.go

```diff
@@ -36,6 +36,9 @@ var TestDefaultConfig = &TestConfiguration{
 	},
 }
 
+var TestNTPSourceSynced = &models.NtpSource{SourceName: "clock.dummy.com", SourceState: models.SourceStateSynced}
```

Comment:
why not under a different test unitls package? i don't think that common is the right place for it. (i don't like common at all)

Reply:
Since this is the de-facto place where we currently have common test related stuff.
I opened a ticket regarding creating a new [testutil package](https://issues.redhat.com/browse/MGMT-3756). This should be moved over there as well.

Reply:
OK thanks